cpu/stm32/include/periph/f2/periph_cpu.h

```diff
@@ -28,7 +28,11 @@ extern "C" {
 /**
  * @brief   Available number of ADC devices
  */
+#if defined(CPU_MODEL_STM32F207ZG)
```

Comment:
Does this work?
```suggestion
#if defined(ADC3)
```

Reply:
I checked both solutions on hardware and both are working. I commit your change. 
BTW. Where in stm32 code is `ADC3` defined?

Reply:
> Where in stm32 code is `ADC3` defined?

Should be in `cpu/stm32/include/vendor`

Reply:
Thanks! And sorry for some old, test code ;)